test/functional/README.md

```diff
@@ -10,7 +10,7 @@ that file and modify to fit your needs.
 
 #### Coverage
 
-Running `test/functional/test_runner.py` with the `--coverage` argument tracks which RPCs are
+Running `build/test/functional/test_runner.py` with the `--coverage` argument tracks which RPCs are
```

Comment:
Per feedback in https://github.com/bitcoin/bitcoin/pull/30859#discussion_r1759115295, begin this sentence with "Assuming the build directory is named `build`, running..."

I agreed with the feedback not to add it where it was already mentioned in other files, but in this file there is no other mention.